pkg/broker/filter/filter_handler.go

```diff
@@ -229,6 +229,10 @@ func (h *Handler) sendEvent(ctx context.Context, headers http.Header, target str
 	defer message.Finish(nil)
 
 	additionalHeaders := utils.PassThroughHeaders(headers)
+
+	//To follow the spec, discussion in: https://github.com/knative/eventing/issues/5771
```

Comment:
I'd explain why we're doing this and a link to the spec more than the issue.
```suggestion
	// The `prefer: reply` HTTP header allows sinks to detect whether replies are acceptable.
```